packages/cli/src/util/projects/connect-git-provider.ts

```diff
@@ -118,3 +120,14 @@ export function parseRepoUrl(originUrl: string): {
     repo,
   };
 }
+
+export function printRemoteUrls(
+  output: Output,
+  remoteUrls: Dictionary<string>
+) {
+  let str = '';
+  for (const [name, url] of Object.entries(remoteUrls)) {
+    str += `  • ${name}: ${chalk.cyan(url)}\n`;
+  }
+  output.print(str);
```

Comment:
Is this method of building a potentially large buffer and calling print once standard? I would assume the overhead of growing the string buffer would be greater than just calling `print()` and writing the output for each entry.